util.py

```diff
@@ -51,43 +50,96 @@ def find_closest_servers(self, location, servers_to_search = None):
                 raise ValueError('Server <' + server + '> latitude/longitude could not be found!')
             server_location = geopy.Point(server_lat_lon[0], server_lat_lon[1])
             server_dict['distance'] = great_circle(item_location, server_location).km
-
             best_servers.append(server_dict)
-
         best_servers.sort(key=self.get_distance_key)
-
     return best_servers
 
+# Converts the local hostname to the simulation ip address
+#
+# params:
+#   local_host: the local hostname
+def convert_to_simulation_ip(local_host):
+    if !os.path.exists(SIMULATION_IP_FILE):
+        return None
+    result = None
+    with open(SERVER_LIST_FILE, 'rb') as server_file, open(SIMULATION_IP_FILE, 'rb') as simulation_ip_file:
+        for line in server_file:
```

Comment:
Nope. The file object is an iterator where each element in the iterator is each line in the file.